pkg/minishift/network/utils.go

```diff
@@ -134,3 +139,41 @@ func OverrideProxyForLocalConnections(proxyAddr string) error {
 	_, err := http.Get("http://localhost")
 	return err
 }
+
+// IsUsingDefaultSwitch returns true if the Default Switch is used before creating the VM
+func IsUsingDefaultSwitch() bool {
+	posh := powershell.New()
+
+	checkIfDefaultSwitchExists := fmt.Sprintf("Get-VMSwitch -Id %s | ForEach-Object { $_.Name }", minishiftConstants.HypervDefaultVirtualSwitchId)
+	stdOut, stdErr, _ := posh.Execute(checkIfDefaultSwitchExists)
+
+	// if the config variable is epmty and default switch exists
+	if VMSwitch == "" && !strings.Contains(stdErr, "Get-VMSwitch") {
```

Comment:
Maybe add something along the lines of:  `// If err contains the command, the execution failed`